src/clock.rs

```diff
@@ -90,6 +101,19 @@ fn setup_main_clock(pmc: &PMC, main_clock: MainClock) -> Hertz {
             // 0 = no prescaling
             0
         }
+        #[cfg(feature = "atsam4n")]
+        MainClock::RcOscillator8Mhz => {
+            switch_main_clock_to_fast_rc_8mhz(pmc);
+
+            // Set up the PLL for 80Mhz operation (8Mhz RC * (12 / 1) = 96Mhz)
+            let multiplier: u16 = 12;
```

Comment:
8 x 25 = 100 MHz

Reply:
I guess what I meant to say earlier is that
8 MHz * 25 / 2 = 100 MHz (which is what the SAM4N asf example code does)